CHANGELOG.next.asciidoc

```diff
@@ -216,6 +216,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 *Functionbeat*
 
 - New options to configure roles and VPC. {pull}11779[11779]
+- Export function templates. {pull}11923[11923]
```

Comment:
Consider this sentence to describe the change:

Export as a file the automation templates used to create functions.